bok_choy/page_object.py

```diff
@@ -399,9 +399,9 @@ def handle_alert(self, confirm=True):
         yield
 
     @unguarded
-    def wait_for_ajax(self):
+    def wait_for_ajax(self, timeout=30):
```

Comment:
Since the default for everything else seems to be 60, we should use that as the default here too.